op_crates/webgpu/lib.rs

```diff
@@ -92,21 +99,23 @@ impl Resource for WebGPUQuerySet {
   }
 }
 
-/// Execute this crates' JS source files.
-pub fn init(isolate: &mut deno_core::JsRuntime) {
-  let files = vec![
-    (
-      "deno:op_crates/webgpu/01_webgpu.js",
-      include_str!("01_webgpu.js"),
+pub fn init(unstable: bool) -> Extension {
+  Extension::with_ops(
+    include_js_files!(
+      prefix "deno:op_crates/webgpu",
+      "01_webgpu.js",
+      "02_idl_types.js",
     ),
-    (
-      "deno:op_crates/webgpu/02_idl_types.js",
-      include_str!("02_idl_types.js"),
-    ),
-  ];
-  for (url, source_code) in files {
-    isolate.execute(url, source_code).unwrap();
-  }
+    declare_webgpu_ops(),
+    Some(Box::new(move |state| {
+      // TODO: check & possibly streamline this
+      // Unstable might be able to be OpMiddleware
+      // let unstable_checker = state.borrow::<super::UnstableChecker>();
+      // let unstable = unstable_checker.unstable;
+      state.put(Unstable(unstable));
```

Comment:
I don't like that this unstable checker is added in `webgpu` instead of in `runtime` crate

Reply:
@bartlomieju `webgpu` (in `main`) already duplicates a good portion of the unstable code into it's own crate. We can remove this with an Unstable middleware in `runtime` (since I think none of `webgpu` ops are conditionally unstable)